lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -660,12 +660,18 @@ func (s *builderServicer) getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, g
 	return minSyncInterval
 }
 
-// getRandomizedSyncInterval returns the interval received from getSyncInterval as seconds and increases it by a random
-// delta in the range [0, getSyncInterval() / 5.0]. Increased sync interval ameliorates the thundering herd effect at
-// the orc8r. Delta is deterministic based on gwKey
+// getRandomizedSyncInterval returns the interval received from getSyncInterval
+// as seconds and increases it by a random delta. Increased sync interval
+// ameliorates the thundering herd effect at the orc8r.
 func (s *builderServicer) getRandomizedSyncInterval(gwKey string, nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs) uint32 {
 	syncInterval := s.getSyncInterval(nwEpc, gwEpc)
+	delta := GetSyncIntervalDelta(gwKey, syncInterval)
+	return syncInterval + delta
+}
 
+// GetSyncIntervalDelta returns a delta of the given sync interval in the range
+// [0, syncInterval / 5.0]. Delta is deterministic based on gwKey.
+func GetSyncIntervalDelta(gwKey string, syncInterval uint32) uint32 {
```

Comment:
1. Have maxDeltaMultiplier be a parameter (where right now it's hard-coded as 1/5), since for the Orc8r-directed resync (this PR) I actually want it to be 1/2
2. Move this method to core Orc8r module, and unrelated to any particular service

```suggestion
func JitterUint32(keyn uint32, key string, maxJitterMultiplier float32) uint32 {
```